lib/chef/cookbook/metadata.rb

```diff
@@ -467,24 +467,24 @@ def gem_requirements_from_array(what, array)
 
       def to_h
         {
-          NAME                   => name,
```

Comment:
best rule ever

Reply:
yeah i think i just lost an argument we had at rally because i'm too lazy to care...

i welcome our new non-aligned overlords...